Main/gyro.ino

```diff
@@ -150,8 +150,18 @@ void detectThrow() {
   This function should detect centrifugal force
 
 */
-
+      // display Euler angles in degrees
 void detectVerticalToHorizontal() {
+   mpu.dmpGetQuaternion(&q, fifoBuffer);
+      mpu.dmpGetEuler(euler, &q);
+     
```

Comment:
Nice use of Euler Values, would be really cool to use them to change the brightness though, and maybe look at the different axis?